Bottom: 9337a14cd7aeb542829cd515e609ca626b6dfd1f
Top:    6d97037bfe51b249c3e1a86746b2a18319f0ef57
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 14:56:00 +0000

Refresh of temp-polygon-mess

---

diff --git a/src/polygon1.c b/src/polygon1.c
index d870f4f..3d236cb 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -741,7 +741,7 @@ edge_label (VNODE * pn, int existing_label)
 
   assert (l);
 
-  fprintf (stderr, "edge_label(), called with an edge on poly %c\n", this_poly);
+  fprintf (stderr, "edge_label(), called with an edge on poly %c\n", l->poly);
   cvc_list_dump (l);
 
   /* Shared edges can be sorted in either order, so need to check l->prev as well */
